### ISSUE TITLE
Uniform sub-command names

### ISSUE THREAD

**Issue by karataliu** [Azure/azure-cli#7855]
**Describe the bug**
In most case, we remove notation 'azure' from product name when design sub command name:
Azure Active Directory -> 'az ad'
Azure CDN -> 'az cdn'

But there are some expectations, we may want to unify the names:

| Component        | Current Name   | Suggested Name  |
| ------------- |:-------------:| -----:|
|Azure Container Registry|acr|cr|
|Azure Container Service|acs|cs|
|Azure Kubernetes Service|aks|ks|
|Azure Media Service|ams|ms/mediaservice|

**To Reproduce**
Run `az`

**Expected behavior**
All the sub-command has a unified rule

**Environment summary**
N/A

**Additional context**
I'd suggest we correct those namings, while adding aliases to existing names to keep backward compatibility.

We'd better have a guideline on how to name sub-commands:
1. Remove 'azure' notation from sub commands
2. When to use full name, when to use acronym

**Comment by tjprescott** [comment]
We have decided not to address this. Service teams would need to buy off on the change and it would be a breaking change for all affected services. At the same time, you can use the alias extension to alias the names to whatever you would like.

**Comment by karataliu** [comment]
@tjprescott  If the cli package itself did renaming and adding alias at the same time, I think there shouldn't be a breaking change.


Personally I've no trouble using any subcommand at all.
The problem is this did bring a non-uniform experience to end-users, and it makes 'bad' examples for future new-added sub-commands.

I'd suggest there to be some additional wiki guides/review process for new added sub-commands, to help ensure we have uniformed experience across all services.

cc @brendandburns as we talked about this a month before.
